ankihub/entry_point.py

```diff
@@ -30,13 +34,19 @@
 from .main.utils import modify_note_type_templates
 from .settings import (
     ADDON_VERSION,
+    ANKI_INT_VERSION,
     ANKI_VERSION,
+    ANKI_VERSION_24_06_00,
     ankihub_db_path,
+    assign_id_to_profile_if_not_exists,
     config,
+    get_anki_profile_id,
     setup_logger,
     setup_profile_data_folder,
 )
 
+CALL_ON_PROFILE_DID_OPEN_ON_MAYBE_AUTO_SYNC = ANKI_INT_VERSION >= ANKI_VERSION_24_06_00
```

Comment:
if the Anki PR gets merged, we probably should limit this to 24.06.x to avoid profile_did_open being called twice.

Reply:
It shouldn't get called twice given the code. Still it's good idea to restrict this behavior to 24.06.xx.
[Only use this on Anki 24.06.xx](https://github.com/AnkiHubSoftware/ankihub_addon/pull/1011/commits/408133e6e7ea8381a8491e20f50ddcc4a6337688)